### PR TITLE
fix: switch color to gchalk

### PR DESCRIPTION
**Pull request by l3gacyb3ta**

### CODE REVIEW

**Comment by l3gacyb3ta** [comment]
@quackduck bish review

File: snode-src/duckcoin.go [hunk 1]
```diff
@@ -149,7 +149,7 @@ func main() {
 		}
 	}
 	address = duckToAddress(pubkey)
-	fmt.Printf("Mining to this address: %s\n", color.HiBlueString(address))
+	fmt.Printf("Mining to this address: %s\n", gchalk.Blue(address))
```
Comment (quackduck) [inline snode-src/duckcoin.go]: Is there a HiBlue option?

**Comment by quackduck** [comment]
Looks good, just make sure all the colors are the "High" version

**Comment by l3gacyb3ta** [comment]
Fixed! Pls merge

File: snode-src/duckcoin.go (hunk 1 above)
Comment (l3gacyb3ta) [reply snode-src/duckcoin.go]: Bam

**Comment by quackduck** [comment]
Okay that fixed it, thanks for contributing, have a great day!

**Comment by l3gacyb3ta** [comment]
u too!

**Pull request merged**